recipes/iwyu/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "include-what-you-use" %}
+{% set version = "0.17" %}
+{% set llvm_version = "13.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ name }}/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 2682727a4462d2d233ac1c03e63b29d606262dd9e54920e7a70dddc25d4cb270
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - python
+    - make
+  host:
+    - {{ compiler('cxx') }}
+    - clangdev {{ llvm_version }}
+    - llvmdev {{ llvm_version }}
+    - llvm {{ llvm_version }}
+    - libclang {{ llvm_version }}
```

Comment:
The build log suggests that this is not needed.
```suggestion
```

Reply:
thanks I will remove that!